mozregression/main.py

```diff
@@ -160,32 +161,41 @@ def _bisect_inbounds(self, good_rev, bad_rev, ensure_good_and_bad=False,
                                  ensure_good_and_bad=ensure_good_and_bad)
         result = self._do_bisect(handler, good_rev, bad_rev, expand=expand)
         if result == Bisection.FINISHED:
-            LOG.info("Oh noes, no (more) inbound revisions :(")
```

Comment:
Hmm. I think we should probably have the log message, now that I think about it. Probably a more meaningful one though -- this isn't an error condition, as is currently implied. I'll revert this change and fix things in a followup.